added x2 variants

Supercookiegaming opened the pull request:
added x2 symbol, footprint and cards
updated to KiCad 9

Supercookiegaming committed:
Added x2

added x2 cards
updated for KiCad 9

Supercookiegaming committed:
Merge pull request #1 from Supercookiegaming/X2

Added x2

lucanastasio commented:
Hi, thank you very much for your contribution. Unfortunately I can't merge the pull request right away as I have to note some issues:

- As far as I know the x2 card is not standard (I checked the rev. 5 specification and I don't have access to rev. 6, if you have more information please let me know), maybe it would be more appropriate having a separate branch for non-standard variants.
- The backup files have not been removed.
- The .kicad_prl files have not been removed.
- The meta/index.htlm file needs an update to KiCad 9 too.

Supercookiegaming commented:
> Hi, thank you very much for your contribution. Unfortunately I can't merge the pull request right away as I have to note some issues:
> 
> * As far as I know the x2 card is not standard (I checked the rev. 5 specification and I don't have access to rev. 6, if you have more information please let me know), maybe it would be more appropriate having a separate branch for non-standard variants.
> * The backup files have not been removed.
> * The .kicad_prl files have not been removed.
> * The meta/index.htlm file needs an update to KiCad 9 too.

I know the x2 is not a standard physical width but the x2 electrical connection width is standard (PCI Express Base Specification Rev 2.1, pg 200, 4.2.4.9) and certain cards do use an [x2 physical connector](https://www.amazon.com/dp/B0B82VRLMM?tag=pcpapi-20&linkCode=ogi&th=1&psc=1). I am working on a project for work that will be using the x2 width so thats why I made this and thought I should update the rest of it.

as for the file stuff I will fix that.

Supercookiegaming committed:
Updated file structure

-Removed Backup Files
-Removed .kicad_prl Files
-Updated meta.html Files
-Moved Example Cards into Templates folder to stop KiCad from giving a template error for the Library folder

Supercookiegaming commented:
> Hi, thank you very much for your contribution. Unfortunately I can't merge the pull request right away as I have to note some issues:
> 
> * As far as I know the x2 card is not standard (I checked the rev. 5 specification and I don't have access to rev. 6, if you have more information please let me know), maybe it would be more appropriate having a separate branch for non-standard variants.
> * The backup files have not been removed.
> * The .kicad_prl files have not been removed.
> * The meta/index.htlm file needs an update to KiCad 9 too.

Hey I updated my fork to include these changes in [my latest commit](https://github.com/lucanastasio/PCIexpress-KiCad/pull/5/commits/cfaae8f64745a205d7c2e29ef8a00193c2cf826f). if there is anything else that needs to be changed let me know.

Supercookiegaming committed:
Added M.2 PCB Footprints

-added M.2 Edge Connector Footprints
-added M.2 Mounting Pad Footprint

Supercookiegaming committed:
Added Symbols for M.2 Module

-Added symbols for all m.2 keys. tx and rx are from the perspective of the m.2 module not the host device.
-updated pcb footprints with better notes

Supercookiegaming committed:
removed extra files

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Merge pull request #2 from Supercookiegaming/M.2

Adding M.2 Symbols and Pinouts

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Setup M.2 Template File Structure

-setup file structure for A, B, E, A+E, B+M templates at all sizes

Supercookiegaming committed:
Updated Template File Structure

-grouped templated by connector type folders

Supercookiegaming committed:
Finished A Key Templates

-Added A Key Templates
-Fixed Name mistakes in symbols
-changed M.2 footprint to have 90 degree thermal relief pads instead of 45 degrees. 45 degrees caused no connections to copper pours.

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Added AC Cap note to A Key Templates

added note about AC coupling caps for diff signals on the a key templates

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Added M Key Templates

-added M key templates for all sized
-corrected M Key symbol pinout
-changed m.2 board thickness for A & M Key Templates

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Added B Key Templates

-added B key Templates for all sizes
-corrected B key symbol pinout

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
added B+M key templates

added B+M key templates in all sizes

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Added E Key Templates

-added E key templates for all sizes
-updated E Key symbol pinout

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Added A+E Key Templates

added A+E key templates for all sizes

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Merge pull request #3 from Supercookiegaming/M.2-Template

M.2 template

Supercookiegaming committed:
Updated M. Footprints

-removed extra drawings from footprints
-added pcb thickness and keepout notes
-fixed keepout area rules from marking the m.2 pads as a violation
-updated all m.2 templates to have new footprints.

Supercookiegaming committed:
Updated A+E Key Footprint

updated A+E Key footprints to remove extra lines and updated their templates

Supercookiegaming committed:
Updating PCIe Symbols

updated the PET/PER pins on the pcie symbols
changed the pin type for power and pci signal pins

Supercookiegaming committed:
Updated PCIe Symbols

fixed pins on all PCIe Symbols input and output are the perspective of the card.

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
updated names on symbols

updated the "Value" field in the m.2 symbols so the name of the connector is shown in your schematic.

updated a few templates with new m.2 symbols

Supercookiegaming committed:
Update README.md

Supercookiegaming committed:
updated all footprints

added allow soldermask bridge option so DRC would not flag errors for different signals existing on the connector

set all pad types to edge connector

set m.2 mounting hole to a solid connection
